mscore/palette/palettetree.cpp

```diff
@@ -482,8 +482,8 @@ static void writePaletteFailed(const QString& path)
 bool PalettePanel::writeToFile(const QString& p) const
       {
       QSet<ImageStoreItem*> images;
-      int n = cells.size();
-      for (int i = 0; i < n; ++i) {
+      size_t n = cells.size();
```

Comment:
Using `auto` is the solution.

Reply:
Hmm, maybe. I'm still way to much of a C person I guess

Reply:
And no, it is not: it then leads to a warning in MinGW